handbook/ce/support.md

```diff
@@ -16,6 +16,22 @@ Because our code is open source, any user can file an issue on our public issue
 - You may not want to discuss your private deployment or configuration, to share your logs, etc.
 - We won't be able to measure (and improve on) our response and resolution times.
 
+### New Customer Welcome Message
+For new customers who have recently signed a contract with us, we want to send a welcome message introducing them to support@sourcegraph.com. The message template is as follows:
```

Comment:
When a new customer first comes onboard (i.e. signs a contract with us for the first time), we like to send them a welcome message from the Support team (and support@sourcegraph.com email address), introducing ourselves and the process for reaching out. The template is as follows: